ibis_mssql/tests/test_basic.py

```diff
@@ -1,10 +1,10 @@
-def test_alltypes_df(sorted_df):
-    pass
-
-
-def test_batting_df(batting_df):
-    pass
-
-
-def test_award_players_df(awards_players_df):
-    pass
+# def test_alltypes_df(sorted_df):
```

Comment:
any reason for commenting that?

I know this is not a robust test, but at least it is testing basic stuff https://github.com/Quansight/ibis-mssql/blob/master/ibis_mssql/tests/conftest.py#L44-L56

Reply:
Hello @xmnlab! Thanks for pointing this out.  I commented it due to the tests failing when I run pytest on the test directory.  I can uncomment them for the next PR.